eyes/eyes.py

```diff
@@ -0,0 +1,379 @@
+import discord
+from discord.ext import commands
+from cogs.utils.dataIO import dataIO
+from cogs.utils import checks
+import red
+from __main__ import send_cmd_help
+import asyncio
+import re
+import time
+import datetime as dt
+from copy import deepcopy
+from random import randint
+from random import choice as randchoice
+
+SETTINGS_PATH    = "data/eyes/settings.json"
+STRF             = "%I:%M%p"
+EMPTY_REGEX_SETS = {"USERS": [], "RAW": []}
+
+
+class C:
+    HEADER    = '\033[95m'
+    OKBLUE    = '\033[94m'
+    OKGREEN   = '\033[92m'
+    WARNING   = '\033[93m'
+    FAIL      = '\033[91m'
+    ENDC      = '\033[0m'
+    BOLD      = '\033[1m'
+    UNDERLINE = '\033[4m'
+    BELL      = '\a'
+
+    def blue(c):
+        return C.OKBLUE + c + C.ENDC
+    def purple(c):
+        return C.HEADER + c + C.ENDC
+    def green(c):
+        return C.OKGREEN + c + C.ENDC
+    def yellow(c):
+        return C.WARNING + c + C.ENDC
+    def red(c):
+        return C.FAIL + c + C.ENDC
+    def bold(c):
+        return C.BOLD + c + C.ENDC
+    def undeline(c):
+        return C.UNDERLINE + c + C.ENDC
+
+
+# REGEX:
+# "{@100406911567949824.display_name}".format(**{'@'+m.id: m for m in server.members})
```

Comment:
this works.
gross for large servers but I kinda want to use it cause it works and it's new.
plus, that means we can do it easily for owner and other users